test/Controllers/ServicesTest.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/*
+ * Resources SDK Controller Test Case
+ */
+
+namespace GonebusyTest\Controllers;
+
+use PHPUnit\Framework\TestCase;
+
+use GonebusyLib\Configuration;
+use GonebusyLib\GonebusyClient;
+use GonebusyLib\Models\CreateServiceBody;
+use GonebusyLib\Models\UpdateServiceByIdBody;
+
+class ServicesTest extends TestCase
+{
+    /**
+     * To contain the GonebusyLib\GonebusyClient
+     */
+    protected $client;
+
+    /**
+     * To contain the GonebusyLib\Controllers\ServicesController
+     */
+    protected $services;
+
+    /**
+     * Create the GoneBusy SDK client for each test.
+     */
+    public function setUp() {
+        $this->client = new GonebusyClient();
+        $this->services = $this->client->getServices();
+    }
+
+
+    /**
+     * Returns arbitrary service data.
+     * @param  string $type Should be 'create' or 'update'.
+     * @return  CreateServiceBody or UpdateServiceByIdBody object with unique data to send to API
+     */
+    private function serviceBody($type) {
+        $rand = rand();
+        switch($type) {
+            case 'create':
+                return new CreateServiceBody(
+                    "description", // REQUIRED
+                    15, // duration REQUIRED XXX 0 not accepted but NULL turns into 0
```

Comment:
Yes, this is correct.  Service must be a multiple of 15min but can not be 0 mins in length.  NULL means you'll set the duration later in case you're not sure now.  I'll make a note to clarify in the documentation. thanks!